src/discordcr/mappings/channel.cr

```diff
@@ -8,7 +8,7 @@ module Discord
       id: {type: UInt64, converter: SnowflakeConverter},
       channel_id: {type: UInt64, converter: SnowflakeConverter},
       author: User,
-      timestamp: {type: Time, converter: Time::Format::ISO_8601_DATE},
+      timestamp: {type: Time, converter: Time::Format.new("%FT%T.%L%:z")},
```

Comment:
This should be extracted into a constant so that it isn't instantiated on every parse.